docs/user/metal/install_ipi.md

```diff
@@ -195,3 +195,80 @@ network configuration, but fully supporting alternative subnets for the
 provisioning network is incomplete.
 
 https://github.com/openshift/installer/issues/2091
+
+## Troubleshooting
+
+General troubleshooting for OpenShift installations can be found
+[here](/docs/user/troubleshooting.md).
+
+### Bootstrap
+
+The bootstrap host is currently a libvirt VM running on the same host as
+the installer. This bootstrap host runs the Ironic services needed to
+provision the control plane. Ironic being available is dependent on
+having successfully downloaded the RHCOS and Ironic agent images. In
+some cases, this may fail, and the installer will report a timeout
+waiting for the Ironic API.
+
+You can view the Ironic logs by sshing to the bootstrap node, and
+examining the logs of the `ironic` service, `journalctl -u ironic`. You
+may also view the logs of the individual containers:
+
+  - `podman logs ipa-downloader`
+  - `podman logs coreos-downloader`
+  - `podman logs ironic`
+  - `podman logs ironic-inspector`
+  - `podman logs ironic-dnsmasq`
+
+To login to the bootstrap VM, you will need to ssh to the host using the
+`core` user, and the SSH key defined in your install config.
+
+The VM obtains an IP address from your DHCP server on the external
+network. When using a development environment with
+[dev-scripts](https://github.com/openshift-metal3/dev-scripts), it uses
+the `baremetal` libvirt network and the IP can be retrieved with `virsh
+net-dhcp-leases baremetal`. If the install is far enough along to have
```

Comment:
The network/bridge is configurable, we could say it defaults to `baremetal` unless an override is specified via `externalBridge` in the install-config